pkgs/misc/base16-shell-preview/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "base16_shell_preview";
```

```suggestion
  pname = "base16-shell-preview";
```

It's preferred to use dashes in the name.
Note you'll have to put the underscored version for `fetchPypi`.